src/test/java/uk/gov/hmcts/reform/ccd/document/am/controller/endpoints/CaseDocumentAmControllerTest.java

```diff
@@ -215,20 +216,6 @@ public void shouldDeleteDocumentByDocumentId() {
         );
     }
 
-    @Test
-    @DisplayName("should throw 403 forbidden when user doesn't have UPDATE permission on requested document")
```

Comment:
why have you removed test case?

Reply:
The idea is to remove all the exception handling from the controller class and confine it in the service layer alone. Thats the reason these exception response code test cases are no more valid.